src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -2269,4 +2269,39 @@ public Object executeScript(String script) {
         return getSeleniumHelper().executeJavascript(statement);
     }
 
+    /**
+     * @param place to check
+     * @return true if content of place is valid (according to validation pattern)
+     */
+    public boolean valueOfIsValid(String place) {
+        T element = getElementToRetrieveValue(place, null);
+        List<T> invalidElements = findAllByCss("input:invalid");
```

Comment:
Don't you need to check whether element is not null?
Furthermore, would this check also work (and be more efficient) using something like `element.findElements(By.cssSelector("input:invalid")).isEmpty()` or does the cssSelector only find nested elements and not element itself?

Reply:
I'll add the null check on the element. I'm not sure what you mean with your second remark. Do you suggest to start with a check if there any invalid elements and if none, no further lookup is needed?